mappings/net/minecraft/item/Item.mapping

```diff
@@ -82,7 +82,17 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_7854 getDefaultStack ()Lnet/minecraft/class_1799;
 	METHOD method_7855 isIn (Lnet/minecraft/class_3494;)Z
 		ARG 1 tag
-	METHOD method_7856 isEffectiveOn (Lnet/minecraft/class_2680;)Z
+	METHOD method_7856 isSuitableFor (Lnet/minecraft/class_2680;)Z
+		COMMENT <p>
+		COMMENT Determines whether this item can be used as a suitable tool for mining the specified block.
+		COMMENT Depending on block implementation, when combined together, the correct item and block may achieve a better mining speed and yield
+		COMMENT drops that would otherwise not be obtained when mining otherwise.
+		COMMENT </p>
+		COMMENT <p>
+		COMMENT Note that this is not the <b>only</b> way to achieve "effectiveness" when mining.
+		COMMENT Other items, such as shears on string, may use their own logic
+		COMMENT and calls to this method might not return a value consistent to this rule for those items.
+		COMMENT </p>
```

Comment:
```suggestion
```

Reply:
Why are you removing the closing <p> tags?